src/features/journey/services/route.service.ts

```
@@ -1,94 +1,115 @@
+            message || 'No routes found between these locations',
+            ErrorCodes.NO_ROUTES_FOUND
+          );
 
-    const routes = await response.json();
+        case ErrorCodes.MISSING_PARAMETERS:
+          throw new RouteError(
+            message || 'Missing required parameters',
+            ErrorCodes.MISSING_PARAMETERS
+          );
+
+        case ErrorCodes.SERVER_ERROR:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)